docs/docs-beta/docs/dagster-plus/deployment/management/tokens/user-tokens.md

```diff
@@ -1,7 +1,41 @@
 ---
-title: User tokens
+title: 'User tokens'
 sidebar_position: 100
 unlisted: true
 ---
 
-{/* TODO move user token content from https://docs.dagster.io/dagster-plus/account/managing-user-agent-tokens */}
\ No newline at end of file
+import ThemedImage from '@theme/ThemedImage';
+
+# Managing user tokens in Dagster+
+
+:::note
+This guide is applicable to Dagster+.
+:::
+
+In this guide, we'll walk you through creating user tokens in Dagster+.
+
+## Managing user tokens
+1. Sign in to your Dagster+ account.
+2. Click the **user menu (your icon) > Organization Settings**.
+3. Click the **Tokens** tab.
+4. Click **+ Create user token**.
+
+After the token is created:
+
+- **To edit a token's description**, click the **pencil icon**.
+- **To view a token**, click **Reveal token**. Clicking on the token value will copy it to the clipboard.
+- **To revoke a token**, click **Revoke**.
+
+To manage tokens for another user, select the user from the **Manage tokens for** dropdown:
+
+<ThemedImage
```

Comment:
Followed the [new format for images](/dagster-plus/deployment/agents) but the image is still showing up as broken

Reply:
Oh yeah, looks like you'll need to copy [this image](https://github.com/dagster-io/dagster/blob/master/docs/next/public/images/dagster-cloud/user-token-management/manage-user-tokens-for.png) to `docs-beta/static/images`. I'll update the CONTRIBUTING with a link to the old images directory.